2021-02-18 20:30:50.092540: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer.so.6'; dlerror: libnvinfer.so.6: cannot open shared object file: No such file or directory
2021-02-18 20:30:50.092853: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer_plugin.so.6'; dlerror: libnvinfer_plugin.so.6: cannot open shared object file: No such file or directory
2021-02-18 20:30:50.092871: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:30] Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
2021-02-18 20:30:53.027176: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2021-02-18 20:30:53.027323: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: UNKNOWN ERROR (303)
2021-02-18 20:30:53.027367: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (c346.oscer.ou.edu): /proc/driver/nvidia/version does not exist
2021-02-18 20:30:53.027615: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2021-02-18 20:30:53.057431: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2297400000 Hz
2021-02-18 20:30:53.058500: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4d1db40 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-18 20:30:53.058523: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
Traceback (most recent call last):
  File "xor_base.py", line 153, in <module>
    display_learning_curve_set("bool_exp")
  File "xor_base.py", line 121, in display_learning_curve_set
    f.save_fig("exp_{}.pdf".format(idx), bbox_inches='tight')
AttributeError: 'Figure' object has no attribute 'save_fig'
